libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json

```diff
@@ -340,6 +395,11 @@
         "comment": "Translate as imperative. Warning email on recipient field in schedule email dialog",
         "limit": 0
     },
+    "close": {
+        "value": "Close",
+        "comment": "",
```

Comment:
I would add something like "caption of a button for closing a dialog window"

Reply:
Good point, didn't realise this may have other meanings when adding it..